lib/tictactoe.rb

```diff
@@ -20,6 +20,16 @@ def initialize (game_io, player1, player2)
   end
 
   def play_game
+    begin
     @game_loop.game_loop
+    @board.new_game
+    end while (play_again?)
+  end
+
+  def play_again?
+    @io.puts_message("Play again (Y/N) : ")
+    input = @io.get_input
+    return true if (input == "y" || input == "Y")
+    false
```

Comment:
Ditto Mike. Also - may be overkill here - but you can take advantage of the ENumerable module if you were to have more choices here (i.e. `["y", "Y"].include?(input)`)